tests/ui/or-patterns/exhaustiveness-unreachable-pattern.rs

```diff
@@ -4,81 +4,107 @@
 fn main() {
```

Comment:
I am not sure what the others think but an argument can be made that every single flagged unreachable pattern is of equal importance within the diagnostic and thus each span should be a primary one (which I think can be realized with `DiagnosticBuilder::set_span` which accepts a `MultiSpan`).

Reply:
Done 👍